pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, fetchFromGitLab , which, go-md2man, coreutils, substituteAll }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.4";
+  pname = "brillo";
+  src = fetchFromGitLab {
+    owner= "cameronnemo";
+    repo= "brillo";
+    rev= "v${version}";
+    sha256 = "0sadsqjxwvmq2jrvq8skvix3jg936k1qsv1x2hw8v9w6craghm99";
+  };
+  makeFlags = [ "PREFIX=$(out)" "AADIR=$(out)/etc/apparmor.d"];
+  nativeBuildInputs = [go-md2man which];
```

Comment:
To be consistent with spacing please change to: `nativeBuildInputs = [ go-md2man which ];`